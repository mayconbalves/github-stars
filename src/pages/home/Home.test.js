import { render, screen } from '@testing-library/react'
import Home from '.'

test('renders home title', () => {
  render(<Home />)
  const title = screen.getByText(/hello world/)
  expect(title).toBeInTheDocument()
})
