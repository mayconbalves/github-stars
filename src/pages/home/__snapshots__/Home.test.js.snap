// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render feedback message without repositories 1`] = `
<div
  class="sc-hHftDr ehMRgB"
  data-testid="spinner"
>
  <div
    class="sc-dmlrTW hKcIyL"
  />
</div>
`;
